multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -40,13 +40,16 @@
 
 class MultiModalPredictor:
     """
-    MultiModalPredictor is a deep learning "model zoo" of model zoos. It can automatically build deep learning models that
-    are suitable for multimodal datasets. You will only need to preprocess the data in the multimodal dataframe format
-    and the MultiModalPredictor can predict the values of one column conditioned on the features from the other columns.
-
-    The prediction can be either classification or regression. The feature columns can contain
-    image paths, text, numerical, and categorical values.
-
+    AutoMM is designed to simplify the fine-tuning of foundation models
+    for downstream applications with just three lines of code.
+    AutoMM seamlessly integrates with popular model zoos such as
+    HuggingFace Transformers (https://github.com/huggingface/transformers),
+    TIMM (https://github.com/huggingface/pytorch-image-models),
+    and MMDetection (https://github.com/open-mmlab/mmdetection),
```

Comment:
Should we make them hyperlinks instead? Seems to be too cluttered in the description [here](http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-3772/9a4dac4/api/autogluon.multimodal.MultiModalPredictor.html).

Reply:
Adding hyperlinks in docstrings seems not straightforward. Feel free to add them if you want.

Reply:
Should have suggested the change in review. I opened a follow up PR for the same #3777.